backend/tests/integration/migrator/test_migrator_view.py

```diff
@@ -130,6 +131,62 @@ def test_get_does_not_return_deleted_records(inspire_app, datadir):
     assert expected_data == response_data
 
 
+def test_get_does_not_return_blacklisted_records(inspire_app, datadir):
+    user = create_user(role=Roles.cataloger.value)
+
+    data = (datadir / "1674997.xml").read_bytes()
+    LegacyRecordsMirrorFactory(
+        recid=1674997,
+        _marcxml=data,
+        collection="HEP",
+        _errors="Error: Least recent error.",
+        valid=False,
+    )
+    data = (datadir / "1674989.xml").read_bytes()
+    LegacyRecordsMirrorFactory(
+        recid=1674989,
+        _marcxml=data,
+        collection="CONFERENCES",
+        _errors="Record: 1674989 has blacklisted pid_type: con is blacklisted",
+        valid=False,
+    )
+    data = (datadir / "1674987.xml").read_bytes()
+    LegacyRecordsMirrorFactory(
+        recid=1674987,
+        _marcxml=data,
+        collection="HEPNAMES",
+        _errors="Error: Most recent error.",
+        valid=False,
+    )
+    with patch.dict(
+        inspire_app.config, {"MIGRATION_PID_TYPE_BLACKLIST": ["con"]}
+    ), inspire_app.test_client() as client:
+        login_user_via_session(client, email=user.email)
+        response = client.get("/migrator/errors", content_type="application/json")
+
+    expected_data = {
+        "data": [
+            {
+                "recid": 1674987,
+                "collection": "HEPNAMES",
+                "valid": False,
+                "error": "Error: Most recent error.",
+            },
+            {
+                "recid": 1674997,
+                "collection": "HEP",
+                "valid": False,
+                "error": "Error: Least recent error.",
+            },
+        ]
+    }
+
+    response_data = json.loads(response.data)
+
+    assert response.status_code == 200
+    assert expected_data == response_data
```

Comment:
```suggestion
    assert expected_data == response.json
```

Reply:
ok, I copy/pasted this from another test, I'll change all tests to use this.